mlir/include/Driver/CompilerDriver.h

```diff
@@ -89,19 +90,17 @@ struct CompilerOptions {
 
 struct CompilerOutput {
     typedef std::unordered_map<Pipeline::Name, std::string> PipelineOutputs;
+    CompilerOutput(int counter = 0) : pipelineCounter(counter) {}
```

Comment:
```suggestion
    CompilerOutput() : pipelineCounter(0) {}
```

Is there any case where we don't want to set it to zero?

Reply:
Yes. We run the compiler driver twice during the compilation. The [first run](https://github.com/PennyLaneAI/catalyst/blob/27443b0c79353043616852732c1e08b93f9603a7/frontend/catalyst/jit.py#L250) might leave some dumps.